spec/lib/config_spec.rb

```diff
@@ -126,7 +152,21 @@ module Fusuma
         end
         it 'should return custom threshold' do
           action_type = 'swipe'
-          expect(Config.threshold(action_type)).to eq 0.5
+          expect(Config.threshold(action_type, event_trigger)).to eq 0.5
+        end
+      end
+
+      context 'when threshold is set on the trigger' do
+        before do
+          allow(YAML).to receive(:load_file).and_return keymap_with_trigger_threshold
```

Comment:
__[RuboCop]__

Metrics/LineLength: Line is too long. [85/80]


<!-- Comment by [Sider](https://sider.review) -->